src/test/java/org/apache/commons/lang3/time/StopWatchTest.java

```diff
@@ -448,4 +470,712 @@ public void testToStringWithMessage() throws InterruptedException {
         final String splitStr = watch.toString();
         assertEquals(12 + MESSAGE.length() + 1, splitStr.length(), "Formatted split string not the correct length");
     }
+
+    @Nested
+    public class SupplierSupportRelatedTests {
+        @Test
+        public void testGet() {
+            final StopWatch watch = StopWatch.create();
+
```

Comment:
Remove blank lines within methods. If you have something interesting to point out, use an inline comment.